Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -3848,8 +3848,10 @@ class ConstructBisector_2 {
     If `l1` and `l2` are parallel, then the bisector is defined as the line
     which has the same direction as `l1`, and which is at the same distance
     from `l1` and `l2`.
-    This function requires that `Kernel::RT` supports the `sqrt()`
-    operation.
+    If `Kernel::FT` is not a model of `FieldWithSqrt`
+    an approximation of the square root will be used in this function,
+    impacting the exactness of the result even with an (exact) multiprecision
+    number type.
```

Comment:
I do not think that sentence can be in the concept, because it documents a particular feature of the implementation of one kernel in CGAL.

Reply:
On the other hand... the sentence is probably true for any implementation that cannot use exact squared roots.

Let's keep it that way.